language/de/ucp.php

```diff
@@ -107,7 +107,7 @@
 	'ACCOUNT_COPPA'					=> 'Dein Benutzerkonto wurde erstellt, muss jedoch erst freigegeben werden. Bitte überprüfe deine E-Mails für weitere Informationen.',
 	'ACCOUNT_EMAIL_CHANGED'			=> 'Dein Benutzerkonto wurde aktualisiert. Jedoch erfordert dieses Board nach der Änderung der E-Mail-Adresse eine erneute Aktivierung. Dazu wurde ein Aktivierungs-Schlüssel an die von dir neu angegebene Adresse geschickt. Bitte überprüfe deine E-Mails für weitere Informationen.',
 	'ACCOUNT_EMAIL_CHANGED_ADMIN'	=> 'Dein Benutzerkonto wurde aktualisiert. Jedoch erfordert dieses Board nach der Änderung der E-Mail-Adresse eine erneute Aktivierung durch einen Administrator. Die Administratoren wurden per E-Mail informiert und du wirst benachrichtigt, sobald dein Benutzerkonto wieder freigeschaltet wurde.',
-	'ACCOUNT_INACTIVE'				=> 'Dein Benutzerkonto wurde erstellt. Du musst es jedoch erst freischalten. Dazu wurde ein Aktivierungs-Schlüssel an die von dir angegebene Adresse geschickt. Bitte überprüfe deine E-Mails für weitere Informationen.',
+	'ACCOUNT_INACTIVE'				=> 'Dein Benutzerkonto wurde erstellt. Du musst es jedoch erst freischalten. Dazu wurde ein Aktivierungs-Schlüssel an die von dir angegebene Adresse geschickt. Bitte überprüfe deine E-Mails für weitere Informationen und überprüfe auch deine Posteingangs-Ordner für Spam-E-Mails. Abhängig von deinem E-Mail-Provider kann es eine Weile dauern, bis du die E-Mail erhälst.',
```

Comment:
anderer Vorschlag (etwas freier):
Bitte überprüfe deine E-Mails für weitere Informationen. Es kann eine Weile dauern, bis dir die E-Mail zugestellt wird. Prüfe bitte ggf. auch die Ordner, in denen Spam- oder Junk-Mails abgelegt werden.

Die Verzögerungen könnten ja auch am E-Mail-Provider des Board-Betreibers liegen.